debvader/debvader.py

```diff
@@ -293,15 +296,32 @@ def detect_objects(field_image, factor=3):
     bkg = sep.Background(r_band_data)
```

Comment:
If you are passing a postage stamp, this estimation can be highly biased. Especially if there is a large object. This could be an issue to detect faint objects. A way to counter that is to do it in 2 passes. One with the "raw" estimate like you do here. You run the detection and output the segmentation map. Then you run it again by masking the objects to have "pure" noise. If you decide to go this way, careful to not count the masked regions otherwise you will underestimate the noise level.